dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -108,13 +129,14 @@ def prep_emissions(self):
 
     def fcst(self):
         dependencies = []
-
         if self.app_config.gefstype in ['gefs-offline']:
             dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic'}
             dependencies.append(rocoto.add_dependency(dep_dict))
         elif self.app_config.gefstype in ['near-real-time']:
             dep_dict = {'type': 'task', 'name': f'{self.run}_stage_ic_mem000'}
             dependencies.append(rocoto.add_dependency(dep_dict))
+            dep_dict = {'type': 'task', 'name': f'{self.run}_gen_control_ic'}
+            dependencies.append(rocoto.add_dependency(dep_dict))
```

Comment:
Similarly, you don't need a dependency on `gefs_stage_ic_mem000` since `gefs_gen_control_ic` has a dependency on `gefs_stage_ic_mem000` already:
```suggestion
            dep_dict = {'type': 'task', 'name': f'{self.run}_gen_control_ic'}
            dependencies.append(rocoto.add_dependency(dep_dict))
```